COMPONENT_VERSION

```diff
@@ -1 +1 @@
-2.4.0
+2.5.0
```

Comment:
Out of curiosity, will these test continue to run for previous releases in any capacity?

Reply:
I believe so, this is just updating the component version that we're using for the main branch (which should be pointing towards 2.5). On the other release branches, the component version should remain consistent with the release that they are building off of.